panther_manager/test/test_manager_bt_node.cpp

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <panther_manager/manager_bt_node.hpp>
+#include "panther_manager/manager_bt_node.hpp"
```

Comment:
This applies to all test files. 
It'd be nice to stick to one convention and include the tested file as the first or not apply this pattern at all. (this varies across our packages)